app/views/data_view.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+from app.models import Donor, Donation, Item
+from django.http import HttpResponse, JsonResponse, HttpResponseBadRequest
+from django.views import View
+import simplejson as json
+import time
+
+@login_required(login_url='/login')
+def aggregate_value(request):
+    '''If request.GET['interval'] is false, return value of all items
+    otherwise, return array of values for given interval
+    '''
+    try:
+        start_date = request.GET['start_date']
+        end_date = request.GET['end_date']
+
+        items = __getModelsGivenInterval('item', start_date, end_date)
+
+        item_date_pairs = __generatePairsWithValue(items)
+        result = {'result': item_date_pairs}
+
+        return JsonResponse(result, status=200)
+    except BaseException:
+        return HttpResponseBadRequest()
+
+
+@login_required(login_url='/login')
+def aggregate_quantity(request):
+    '''Return aggregate quantity of given model for given time interval.'''
+    try:
+        model = request.GET['model']
+        start_date = request.GET['start_date']
+        end_date = request.GET['end_date']
+
+        items = __getModelsGivenInterval(model, start_date, end_date)
+
+        aggregated_quantity = __generatePairsWithQuantity(items)
+        result = {'result': aggregated_quantity}
+        
+        return JsonResponse(result, status=200)
+    except BaseException:
+        return HttpResponseBadRequest()
+
+'''
+Private
+'''
+def __getModelsGivenInterval(model, start_date, end_date):
```

Comment:
You are technically getting a queryset (objects) not model. It should be named `__getObjectsGivenInterval` or `__getQuerysetGivenInterval`